.github/workflows/release.yml

```diff
@@ -15,8 +15,11 @@ on:
     tags:
       - 'v*'
 env:
-  GO_VERSION: 1.21
+  # This should be the latest major release of Go.
+  GO_VERSION: 1.23
```

Comment:
Should this be `1.22` to match `go.mod`? Before we had `1.21` in both this file and `go.mod` so wasn't sure if we want them to match again this time.